monai/metrics/hausdorff_distance.py

```diff
@@ -18,18 +18,20 @@
 from monai.metrics.utils import do_metric_reduction, get_mask_edges, get_surface_distance, ignore_background
 from monai.utils import MetricReduction
 
+from .metric import Metric
+
 __all__ = ["HausdorffDistanceMetric", "compute_hausdorff_distance", "compute_percent_hausdorff_distance"]
 
 
-class HausdorffDistanceMetric:
+class HausdorffDistanceMetric(Metric):
     """
     Compute Hausdorff Distance between two tensors. It can support both multi-classes and multi-labels tasks.
     It supports both directed and non-directed Hausdorff distance calculation. In addition, specify the `percentile`
     parameter can get the percentile of the distance.
     Input `y_pred` (BNHW[D] where N is number of classes) is compared with ground truth `y` (BNHW[D]).
     `y_preds` is expected to have binarized predictions and `y` should be in one-hot format.
     You can use suitable transforms in ``monai.transforms.post`` first to achieve binarized values.
-
+    `y_preds` and `y` can also be a list of Tensor with shape: [CHW[D]].
```

Comment:
`y_pred` and `y` can be a list of `channel-first` Tensor (CHW[D]) or a `batch-first` Tensor (BCHW[D]).

Reply:
updated in the latest commit.
Thanks.